session/session.go

```diff
@@ -1064,6 +1065,14 @@ func (s *session) executeStatement(ctx context.Context, connID uint64, stmtNode
 	return recordSets, nil
 }
 
+func (s *session) ExecuteInternal(ctx context.Context, sql string) (recordSets []sqlexec.RecordSet, err error) {
```

Comment:
Is it possible to call `ExecRestrictedSQL` instead?

Reply:
Let me take a look.

Reply:
It's hard to do this because the binding handler contains some session information outside. If we change it to `ExecRestrictedSQL`, there some errors will occur due to the session info miss.